README.RDAP.md

```diff
@@ -61,3 +61,10 @@ The RIPE database doesn't contain any forward domain objects, consequently a nam
 Only "mnt-by:" Maintainers are Listed as Registrants
 -----------------------------------------------------
 Only maintainers referenced in "mnt-by:" attributes will be listed as Registrants in responses.
+
+IANA delegated objects will return a 404
+-----------------------------------------------------
+There must be a placeholder with "administrative" status for the IANA-delegated object.
```

Comment:
If the prefix is either delegated but unallocated or only partially delegated to the RIPE region, then a 404 is returned. An object with "administrative" status is never returned.